package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "typings": "index.d.ts",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "mocha *.test.js"
```

Comment:
If you name the file `test.js` instead of `index.test.js` you don't have to explicitly state the name here, and it will be aligned more closely with how the tests files are usually named by following the default settings.